app/index.js

```diff
@@ -76,6 +75,7 @@ var FluxGenerator = yeoman.generators.Base.extend({
     this.composeWith('flux:dispatcher', {args: ['AppDispatcher']});
     this.composeWith('flux:store', {args: ['DataStore']});
     this.composeWith('flux:action', {args: ['DataActionCreators']});
+    this.composeWith('flux:component', {args: ['App']});
```

Comment:
Based on the comment above, should this be defaulted to something other than App since that's taken?